Detectors/EMCAL/calibration/testWorkflow/EMCALChannelCalibratorSpec.h

```diff
@@ -227,6 +240,10 @@ DataProcessorSpec getEMCALChannelCalibDeviceSpec()
   std::vector<InputSpec> inputs;
   inputs.emplace_back(device::getCellBinding(), o2::header::gDataOriginEMC, "CELLS", 0, o2::framework::Lifetime::Timeframe);
   inputs.emplace_back(device::getCellTriggerRecordBinding(), o2::header::gDataOriginEMC, "CELLSTRGR", 0, o2::framework::Lifetime::Timeframe);
+  // for loading the channelCalibParams from the ccdb
+  if (loadCalibParamsFromCCDB) {
```

Comment:
it would be enough to call this only once, since params don't change during the run

Reply:
Okay that makes sense. I will add that!